docs/shipping/Code/dotnet.md

```diff
@@ -306,53 +306,72 @@ namespace dotnet_log4net
 ```
 
 ##### Serverless platforms
-If you’re using a serverless function, you’ll need to call the appender's flush methods at the end of the function run to make sure the logs are sent before the function finishes its execution.
+If you’re using a serverless function, you’ll need to call the appender's flush methods at the end of the function run to make sure the logs are sent before the function finishes its execution. You’ll also need to create a static appender in the Startup.cs file so each invocation will use the same appender. 
```

Comment:
Serverless functions can be found in all cloud providers. The Startup.cs script is specific for Azure Function, so I think you need to have a sub-section in serverless platform specificaly for Azure Function